samples/utilities/checkPrereqs.sh

```diff
@@ -0,0 +1,582 @@
+#/bin/sh
+
```

Comment:
Do we need some type of license - apache or ?

Reply:
License in repo governs this script.